en/docs/integrations/integration-concepts.md

```diff
@@ -1,57 +1,55 @@
+ - **GitHub**: Connects your service to GitHub API to perform operations in GitHub, such as creating an issue, creating a pull request, etc. 
+ - **Gmail**: Connects your service to Gmail API to perform email operations such as sending an email, receiving an email, etc.
+ - **Google Calendar**: Connects your service to Google Calendar API to perform operations in Google Calendar such as creating an event, deleting an event, etc.
+ - **Google Sheets**: Connects your service to Google Sheets API to perform operations in Google Sheets such as reading data, formatting data, etc.
+ - **Salesforce**: Connects your service to Salesforce API to perform Salesforce operations such as obtaining organization information, creating a record, etc.
+ - **NetSuite**: Connects your service to Netsuite API to perform Netsuite operations such as creating a record instance, creating a sublist, etc. 
 
 ## Statement
-A statement in Choreo integrations is a syntactic unit that specifies the action to be carried out in an integration. 
+
+A statement is a collection of code fragments that specifies the actions to be carried out by a Choreo integration. Choreo statements can also be composed of expressions. These expressions are written based on the [Ballerina Expression Syntax](#ballerina-expression-syntax).
 
 Choreo supports the following statements:
 
- - **Log**: To add an entry to the log at runtime. You can either add an **Info** log or an **Error** log depending on your requirement.   
- - **Variable**: To declare a variable of a preferred type depending on your requirement.
- - **If**: To perform a conditional check and evaluate whether a statement is true or false.   
- - **ForEach**: To iterate over a list of items to perform control flow. 
- - **While**: To iterate over a given condition to perform control flow. 
- - **Return**: To end the ongoing execution of an integration and return control to the caller.
- - **Custom**: To specify a custom statement depending on your requirement.
+ - **Log**: Logs an event with an information statement or an error that occurs in your service.    
+ - **Variable**: Declares a variable and a value of a preferred data type (string, integer, etc.) to use later in the logical process of the service.
+ - **If**: Specifies two blocks of logical components so that the system can decide which block to execute based on whether the provided condition is true or false.   
+ - **ForEach**: Includes a control flow statement in your service to iterate over a list of items.
+ - **While**: Continuously executes a block of statements in a loop as long as the specified condition is TRUE. 
+ - **Respond**: Sends a specific response back to the client. For example, if the request was processed successfully,  responding to the client with an `HTTP_OK`.
+ - **Custom**: Is used to write a single or a multiline code snippet in your service.
```

Comment:
Why we are following a different style here starting with `Is used..`?

Reply:
Because adding a custom statement doesn't directly imply what you are doing. I felt writing it like this will make it clear to the reader. Any suggestions?

Reply:
How about `Allows to write a single or multiline code snippet in your service` ?